go-controller/pkg/node/controllers/egressip/egressip.go

```diff
@@ -1321,7 +1321,10 @@ func overwriteRoutesTableID(routes []netlink.Route, tableID int) {
 }
 
 func getRouteTableID(ifIndex int) int {
-	return ifIndex + routingTableIDStart
+	if !util.IsNetworkSegmentationSupportEnabled() {
+		return ifIndex + routingTableIDStart
```

Comment:
let's just change routingTableIDStart to 30K from 1000, so that its always consistent cc @martinkennelly wdyt?